### PR TITLE
fix: better errors from space wizard

### LINKED ISSUE #360: Create Space 'take me' link broken
When I create a new space and it succeeds the modal changes to the "how to get started" view. There is a button at the bottom that says "no thanks, just take me to the [space name]". That button doesn't take me anywhere and I just stay on the modal.

### PR DESCRIPTION
**Pull request by pmuir**
Also fixes #360

### CODE REVIEW

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: dc7b575a20d7f7625934ec15c5b4334cda317ded
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/fabric8io/fabric8-ui

---
This message was auto-generated by https://gitcop.com

File: src/app/shared/runtime-console/space-namespace.service.ts [hunk 1]
```diff
@@ -39,7 +39,7 @@ export class SpaceNamespaceService {
       (namespace, loading) => namespace
     )
       .switchMap(namespace => this.configMapService
-        .list(namespace)
+        .list(namespace+'1')
```
Comment (joshuawilson) [inline src/app/shared/runtime-console/space-namespace.service.ts]: What does this do?

Comment (pmuir) [inline src/app/shared/runtime-console/space-namespace.service.ts]: OOOOPS that was my fault injection. I need to remove that.

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 7234d43c49b64af55fd55ded9d6e90aec4728558
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/fabric8io/fabric8-ui

---
This message was auto-generated by https://gitcop.com

**Comment by pmuir** [comment]
@joshuawilson error removed :-D

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: abd70614034b102ed01792c7eeb2c70e15cc3e7a
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/fabric8io/fabric8-ui

---
This message was auto-generated by https://gitcop.com

**Pull request merged**